### PR TITLE
Bump pyscript version and improve template

### PR DESCRIPTION
**Pull request by FabioRosado**
Changes to the template follow the one we have in PSDC

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -237,7 +237,7 @@ def test_wrap_pyscript_version(
     )
     css_version_str = (
         '<link rel="stylesheet" href="https://pyscript.net/releases/'
-        f'{expected_version}/core.css">'
+        f'{expected_version}/core.css" />'
```
Comment (WebReflection) [inline tests/test_cli.py:240]: for HTML that ` />` is not needed, completely useless in the wild

File: src/pyscript/templates/basic.html [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
-<head>
+  <head>
     <title>{{ title }}</title>
-    <link rel="stylesheet" href="https://pyscript.net/releases/{{ pyscript_version }}/core.css">
+
+    <!-- Recommended meta tags -->
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+
+    <link rel="stylesheet" href="https://pyscript.net/releases/{{ pyscript_version }}/core.css" />
```
Comment (WebReflection) [inline src/pyscript/templates/basic.html:10]: for HTML that ` />` is not needed, completely useless in the wild

File: src/pyscript/templates/basic.html [hunk 3]
```diff
@@ -1,11 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <title>{{ title }}</title>
-    <link rel="stylesheet" href="https://pyscript.net/releases/{{ pyscript_version }}/core.css">
+
+    <!-- Recommended meta tags -->
+    <meta charset="UTF-8" />
```
Comment (WebReflection) [inline src/pyscript/templates/basic.html:7]: for HTML that ` />` is not needed, completely useless in the wild

**Comment by WebReflection** [comment]
to expand in one place about my comments, void elements never need ` />` as these are ... well, **void** elements, no XML like self-closing tag needed, as that's simply **not supported in HTML5**, so you could save some char and not propose a template that no IDE would generate automatically as `meta`, `link`, among others, don't need to be closed because can't accept child nodes: https://developer.mozilla.org/en-US/docs/Glossary/Void_element

**Comment by FabioRosado** [comment]
Done thanks 😄

**Review by fpliger — APPROVED** [review]
LGTM

**Pull request merged**